src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala

```diff
@@ -0,0 +1,31 @@
+
+package org.broadinstitute.dsde.agora.server.dataaccess.mongo
+
+import com.mongodb.casbah.Imports._
+import com.mongodb.casbah.MongoCollection
+import com.mongodb.casbah.query.Imports
+import com.novus.salat._
+import com.novus.salat.global._
+import org.broadinstitute.dsde.agora.server.dataaccess.AgoraDao
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+class AgoraMongoDao(collection: MongoCollection) extends AgoraDao {
+
+  def findPayloadByRegex(regex: String): Seq[AgoraEntity] = {
+    find("payload" $regex regex)
+  }
+
+  def findByName(name: String): Seq[AgoraEntity] = {
```

Comment:
Will we want to do a find by name/namespace?

Also, for the present story, do we want findById?